store.go

```diff
@@ -2550,6 +2551,14 @@ func (s *store) Lookup(name string) (string, error) {
 }
 
 func (s *store) DeleteLayer(id string) error {
+	var cleanUp tempdir.CleanupTempDirFunc
+	defer func() {
+		if cleanUp != nil {
+			if err := cleanUp(); err != nil {
+				logrus.Errorf("Error cleaning up after deleting layer %q: %v", id, err)
```

Comment:
- This error should definitely be reported to the caller.
- (The `Error` prefix will probably be unnecessary if this is wrapped.)
- Probably `%w`